app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -129,6 +141,46 @@ class DragAndDropSortInteractionViewModel private constructor(
       this@DragAndDropSortInteractionViewModel.writtenTranslationContext
   }.build()
 
+  override fun getRawUserAnswer(): RawUserAnswer = RawUserAnswer.newBuilder().apply {
```

Comment:
Able to retain state properly for DragAndDrop interaction but for DragAndDropSort, I am unable to link and unlink elements as it required recyclerview adapter. For reference you can watch video below. 


https://user-images.githubusercontent.com/43074241/199037552-0931c64f-7e6d-4797-8e15-f51d48c53169.mp4

Reply:
We chatted about this during the meeting this morning, and I left a similar comment elsewhere on the PR to suggest using content IDs, instead, as that should simplify things here.

Reply:
Resolving this now as this issue is solved.